testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -754,6 +754,29 @@ QUnit.module("options changing", moduleConfig, () => {
         }).trigger(dxPointerDown);
     });
 
+    QUnit.test("tap on clear button should not raise onValueChange event (T812448)", (assert) => {
+        const valueChangeStub = sinon.stub();
+
+        const $element = $("#texteditor").dxTextEditor({
+            showClearButton: true,
+            onValueChanged: valueChangeStub
+        });
+        const $clearButton = $element.find(CLEAR_BUTTON_SELECTOR).eq(0);
+        const kb = this.keyboard;
+        kb.type("clear");
+        assert.equal(this.input.val(), "clear", "texteditor has correct value");
+
+        const dxPointerDown = $.Event("dxpointerdown");
+        dxPointerDown.pointerType = "touch";
+        $clearButton.on("dxpointerdown", e => {
+            assert.equal(this.input.val(), "", "texteditor is empty");
+        }).trigger(dxPointerDown);
+
+        kb.type("change")
+            .change();
+        assert.equal(valueChangeStub.calledOnce, 1, "onValueChanged was called once");
```

Comment:
calledOnce: **boolean**
```suggestion
        assert.ok(valueChangeStub.calledOnce, "onValueChanged was called once");
```